TokenCreation.sol

```diff
@@ -112,7 +112,8 @@ contract TokenCreation is TokenCreationInterface, Token {
     function refund() noEther {
         if (now > closingTime && !isFueled) {
             // Get extraBalance - will only succeed when called for the first time
-            extraBalance.payOut(address(this), extraBalance.accumulatedInput());
+            if (extraBalance.balance >= extraBalance.accumulatedInput())
```

Comment:
This if check is just to save some gas by not calling a no-effect function again, right?